spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -1,20 +1,59 @@
 # frozen_string_literal: true
 
+require 'rails_helper'
+
 RSpec.describe Api::V1::KeywordsController, type: :request do
   describe 'GET#index' do
-    it 'returns success status' do
-      get api_v1_keywords_path, headers: create_token_header
+    context 'given a logged in user' do
+      context 'given the user query query their own keyword' do
+        it 'returns success status' do
+          user = Fabricate(:user)
+          Fabricate.times(3, :keyword, user: user)
 
-      expect(response).to have_http_status(:success)
-    end
+          get api_v1_keywords_path, headers: create_token_header(user)
+
+          expect(response).to have_http_status(:success)
+        end
+
+        it 'returns three items' do
+          user = Fabricate(:user)
+          Fabricate.times(3, :keyword, user: user)
+
+          get api_v1_keywords_path, headers: create_token_header(user)
+
+          response_body = JSON.parse(response.body, symbolize_names: true)
+          expect(response_body[:data].count).to eq 3
+        end
+      end
+
+      context 'given the user has no keyword' do
+        it 'returns success status' do
+          user = Fabricate(:user)
+          Fabricate(:keyword, user: user)
```

Comment:
```suggestion
          _another_user_keyword = Fabricate(:keyword)
```

(ensure your keyword fabricator Fabricates a user when not provided)